projects/plugins/jetpack/_inc/jetpack-server-sandbox.php

```diff
@@ -1,38 +1,7 @@
 <?php
 /**
- * This feature is only useful for Automattic developers.
- * It configures Jetpack to talk to staging/sandbox servers
- * on WordPress.com instead of production servers.
+ * The feature formerly here is now found in `Automattic\Jetpack\Connection\Server_Sandbox`
+ * in the jetpack-connection package.
```

Comment:
Do we need to keep this file? If so, maybe we should put a "since" mention in the comment so 2 years down the road it's clear it's safe to remove.

Reply:
> Do we need to keep this file?

Not sure.

> If so, maybe we should put a "since" mention in the comment so 2 years down the road it's clear it's safe to remove.

Good idea, I'll do that.